docker-compose.yml

```diff
@@ -0,0 +1,34 @@
+# version: '1'
+services:
+  app:
+    build:
+      context: .
+    ports:
+      - "4000:8000"
+    volumes:
+      - ./:/app
+      - my_media:/files/media
+    command: >
+      sh -c "python manage.py wait_for_db &&
+            python manage.py makemigrations &&
```

Comment:
It's not good idea make migrations each time when you start docker. You should make migrations manually. Delete it please from docker compose